### PR TITLE
chore(changelog): add changelog Makefile

### PR DESCRIPTION
**Pull request by vm-001**
Summary

Add a changelog Makefile for easy using

### CODE REVIEW

**Comment by vm-001** [comment]
@outsinre Would you recommend to put the changelog validation in this PR, or you prefer to make it in a separate PR?

File: changelog/Makefile [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.PHONY: changelog-generate changelog-regenerate
+
+CHANGELOG_VERSION=1.0.0
+CHANGELOG_FOLDER=unreleased
+changelog-generate:
```
Comment (outsinre) [inline changelog/Makefile:5]: Can we simplify the command name?

```suggestion
generate:
```

File: changelog/Makefile [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+			--changelog-paths $(CHANGELOG_FOLDER)/kong-manager \
+			--title Kong-Manager \
+			--github-issue-repo Kong/kong-manager \
+			--github-api-repo Kong/kong \
+			>> $(CHANGELOG_FOLDER)/$(CHANGELOG_VERSION).md; \
+    fi
+	@echo "Successfully genreate changelog $(CHANGELOG_FOLDER)/$(CHANGELOG_VERSION).md"
+
+CHANGELOG_VERSION=1.0.0
+changelog-regenerate:
```
Comment (outsinre) [inline changelog/Makefile:29]: Can we simplify the command name?

```suggestion
regenerate:
```

**Comment by outsinre** [comment]
https://github.com/Kong/kong/pull/12610.

**Pull request closed without merging**